src/Aspire.Dashboard/Resources/Columns.resx

```diff
@@ -189,4 +189,7 @@
   <data name="EndpointsColumnDisplayOverflowTitle" xml:space="preserve">
     <value>Endpoints</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="PersistentContainerIconTooltip" xml:space="preserve">
+    <value>This resource is persistent and won't be stopped when the app host is shutdown.</value>
```

Comment:
This should be 

```suggestion
    <value>This resource is persistent and won't be stopped when the app host is shut down.</value>
```

(xlf files need regenerating, so can't just merge this on github.com)